server/mlabns/conf/tools.csv

```diff
@@ -1,5 +1,6 @@
 tool_id,slice_id,http_port,show_tool_extra
 ndt,iupui_ndt,7123,False
+ndt_ssl,iupui_ndt,7123,False
```

Comment:
I don't think fakewww behavior should be different between NDT and NDT-SSL. We might one day offer a fakewww with TLS support for people who want to embed the NDT websockets frontend, but I see that as independent from this change.